charts/cortex/templates/ingester/ingester-svc.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.topologyAware.enabled }}
```

Comment:
I would keep this as the non-headless service and put the "if-else" into `ingester-svc-headless.yaml`